aten/src/ATen/native/native_functions.yaml

```diff
@@ -380,7 +380,8 @@
     CompositeExplicitAutograd: _conj_physical
     SparseCsrCPU, SparseCsrCUDA: conj_physical_sparse_csr
 
-- func: conj_physical(Tensor self) -> Tensor
+# TODO: should be -> Tensor(a|b) but breaks python alias parser
```

Comment:
Just... add support for this syntax? It shouldn't be hard. Alias parsing is at tools/codegen/model.py